src/lib/viewers/controls/settings/SettingsDropdown.tsx

```diff
@@ -7,20 +7,26 @@ import useClickOutside from '../hooks/useClickOutside';
 import { decodeKeydown } from '../../../util';
 import './SettingsDropdown.scss';
 
-export type ListItem = {
+export type ListItem<V> = {
     label: string;
-    value: string;
+    value: V;
 };
 
-export type Props = {
+export type Props<V> = {
```

Comment:
I think we can use `export type Props<V = string>` here to avoid having to specify types in our tests. Further, should we have an `extends` here to only allow certain types, such as boolean/number/string?